code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/OptimizeExtension.java

```diff
@@ -859,6 +886,19 @@ private Event createResponseEventWithError(final Event event, final AdobeError e
                 .build();
     }
 
+    private Event createResponseEventWithError(final Event event, final AEPOptimizeError error) {
+        final Map<String, Object> eventData = new HashMap<>();
+        eventData.put(OptimizeConstants.EventDataKeys.RESPONSE_ERROR, error);
```

Comment:
note: This error is not passed back to the customer in `Optimize.java`. Guessing it would come in the next PR